### PR TITLE
Updating email to use the latest python sdk

### PR DESCRIPTION
**Pull request by anmolbohra97**
---
Updating email in communication extension to use the latest GA version of email python sdk.

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/communication/azext_communication/manual/_help.py [hunk 1]
```diff
@@ -564,25 +564,11 @@
...
-helps['communication email status'] = """
-    type: group
-    short-summary: Commands to get the status of emails previously sent using Azure Communication Services Email service.
-"""
-
 helps['communication email send'] = """
     type: command
-    short-summary: "Send an email."
+    short-summary: "Send an email and get final status."
     examples:
```
Comment (apattath) [inline src/communication/azext_communication/manual/_help.py:570]: should the example also contain a sample to show how to get the operation status?

File: src/communication/azext_communication/manual/_params.py [hunk 2]
```diff
@@ -255,10 +255,10 @@ def _load_email_arguments(self):
         c.argument('subject', options_list=['--subject'], type=str, help='Subject of the email message.')
         c.argument('text', options_list=['--text'], type=str, help='Plain text version of the email message. Optional.')
         c.argument('html', options_list=['--html'], type=str, help='Html version of the email message. Optional.')
-        c.argument('importance', options_list=['--importance'], arg_type=get_enum_type(['normal', 'low', 'high']),
-                   help='The importance type for the email. Known values are: high,'
-                   ' normal, and low. Default is normal. Optional')
         c.argument('recipients_to', options_list=['--to'], nargs='+', help='Recepients email addresses.')
+        c.argument('importance', options_list=['--importance'], arg_type=get_enum_type(['1','2','3','4','5']),
+                   help='The importance type for the email. Possible values include: 1(High), 2(High),'
+                   '3(Normal), 4(Low), 5(Low). Optional.')
```
Comment (apattath) [inline src/communication/azext_communication/manual/_params.py:261]: We should only provide 3 possible values to customer - high, normal and low (as it used to be before). In the implementation, you should convert it to a number from 1 to 5 and send it in the custom header.

File: src/communication/azext_communication/manual/_params.py [hunk 3]
```diff
@@ -272,7 +272,3 @@ def _load_email_arguments(self):
                    ' Required for each attachment. Known values are: avi, bmp, doc, docm,'
                    ' docx, gif, jpeg, mp3, one, pdf, png, ppsm, ppsx, ppt, pptm, pptx,'
                    ' pub, rpmsg, rtf, tif, txt, vsd, wav, wma, xls, xlsb, xlsm, and xlsx')
-
-    with self.argument_context('communication email status get') as c:
-        c.argument('message_id', options_list=['--message-id'], type=str,
-                   help='System generated message id (GUID) returned from a previous call to send email')
```
Comment (apattath) [inline src/communication/azext_communication/manual/_params.py:278]: is the get operation status not needed?

File: src/communication/azext_communication/manual/custom.py [hunk 4]
```diff
@@ -285,85 +285,96 @@ def communication_rooms_remove_participants(client, room_id, participants):
...
+                ],
+            "attachments": None if attachments_list is None else
+                [
+                    json.loads(attachment) for attachment in attachments_list
+                ],
+            "senderAddress": sender,
+            "userEngagementTrackingDisabled": disable_tracking,
+            "headers": None if importance is None else
+                {
+                    "x-priority": importance
```
Comment (apattath) [inline src/communication/azext_communication/manual/custom.py:373]: Here you should take customer input and convert it into a value between 1-5. See previous related comment.

**Review by apattath — CHANGES_REQUESTED** [review]

File: src/communication/azext_communication/manual/custom.py (hunk 4 above)
Comment (anmolbohra97) [reply src/communication/azext_communication/manual/custom.py:373]: Got it

File: src/communication/azext_communication/manual/_params.py (hunk 3 above)
Comment (anmolbohra97) [reply src/communication/azext_communication/manual/_params.py:278]: With LRO, we no longer have a separate call to get message status.

File: src/communication/azext_communication/manual/_help.py (hunk 1 above)
Comment (anmolbohra97) [reply src/communication/azext_communication/manual/_help.py:570]: The Send call also gets the final status

File: src/communication/azext_communication/manual/_params.py (hunk 2 above)
Comment (anmolbohra97) [reply src/communication/azext_communication/manual/_params.py:261]: Makes sense. Updating.

File: src/communication/azext_communication/manual/_params.py [hunk 5]
```diff
@@ -256,8 +256,15 @@ def _load_email_arguments(self):
...
         c.argument('recipients_to', options_list=['--to'], nargs='+', help='Recepients email addresses.')
+        c.argument('importance', options_list=['--importance'], arg_type=get_enum_type(['normal', 'low', 'high']),
+                   help='The importance type for the email. Known values are: high,'
+                   ' normal, and low. Default is normal. Optional')
         c.argument('recipients_cc', options_list=['--cc'], nargs='+', help='Carbon copy email addresses.')
         c.argument('recipients_bcc', options_list=['--bcc'], nargs='+', help='Blind carbon copy email addresses.')
+        c.argument('reply_to', options_list=['--reply-to'], type=str, help='Reply-to email address. Optional.')
+        c.argument('disable_tracking', options_list=['--disable-tracking'], arg_type=get_three_state_flag(),
+                   help='Indicates whether user engagement tracking should be disabled for this request if'
+                   'the resource-level user engagement tracking setting was already enabled. Optional.')
```
Comment (apattath) [inline src/communication/azext_communication/manual/_params.py]: Suggestion to re-write the help description as below for more clarity:
"Indicates whether user engagement tracking should be disabled for this specific request. This is only applicable if the resource-level user engagement tracking setting was already enabled in control plane."

**Review by jorge-beauregard — APPROVED** [review]
LGTM. Please fix the lint issues in the manual/custom file

**Comment by anmolbohra97** [comment]
> LGTM. Please fix the lint issues in the manual/custom file

@jorge-beauregard  looks like it has succeeded now. Could you merge this change now?

**Pull request merged**